docs/_includes/subscribe.html

```diff
@@ -35,6 +35,19 @@ <h3 class="section-subheading text-muted">{{ site.data.sitetext.subscribe.text |
           </form>
 
         </div>
+
+      </div>
+
+      
+
+    </div>
+    <!--Mailing List-->
+    <div class="mailing-container">
+      <h1>Mailing Lists</h1>
+      <p>Engage in the Service Mesh Performance project. Join any of our mailing lists.</p>
+      <div class="mailing-btn-container">
+          <a href="https://groups.google.com/a/smp-spec.io/d/forum/community" class="btn btn-primary btn-xl text-uppercase ">User</a>
+          <a href="mailto:maintainers@smp-spec.io" class="btn btn-primary btn-xl text-uppercase ">Maintainer</a>
```

Comment:
```suggestion
          <a href="mailto:maintainers@smp-spec.io" class="btn btn-primary btn-xl text-uppercase ">Maintainers</a>
```

Reply:
I will fix these.

Reply:
Thanks. You can just click to commit these suggestions or batch commit the suggestions.

Reply:
yeah sure, I will commit these changes.  Are we going with Dark Black bg color or the same as newsletter?